content/content/string_formatting.md

```diff
@@ -0,0 +1,96 @@
+---
+title: String Formatting
+---
+
+# String Formatting
+
+The `&` operator can be used for simple string concatenation:
+
+``` nimrod
```

Comment:
It is called Nim.

Reply:
Hmm, I was following the convention of the rest of the doc pages, which are all set to "nimrod". We should probably do a full search/replace to just nim.